docker/ops.go

```diff
@@ -54,55 +56,11 @@ func parse_uri(s string) (Reference, error) {
 	return Reference{Path: path, Domain: domain, Tag: tag, Digest: digest}, err
 }
 
-func generate_sbom(message *babashka.Message, image string, username string, password string) error {
```

Comment:
Why are we removing things?

Reply:
It is the quickest solution. See other comment.

Reply:
Or rather we can't use it without dealing with dependencies.